reverb/cc/platform/default/repo.bzl

```diff
@@ -98,6 +103,10 @@ def _find_python_solib_path(repo_ctx):
         fail("Could not locate python shared library path:\n{}"
             .format(exec_result.stderr))
     solib_dir = exec_result.stdout.splitlines()[-1]
+    if is_darwin(repo_ctx):
+        basename = "lib{}m.dylib".format(version)
```

Comment:
@lifeiteng I was trying to compile on macOS. I needed to change `lib{}m.dylib` to `lib{}.dylib` though. This seems to be a detail in the Python build https://stackoverflow.com/questions/32544974/naming-convention-what-does-the-m-mean-in-libpython3-5m-dylib.

Not sure about the best way to fix this though.